sas9.4-viya/azuredeploy.json

```diff
@@ -0,0 +1,3683 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "SAS Application Name": {
+            "minLength": 2,
+            "maxLength": 5,
+            "type": "String",

```

Comment:
add defaultValues for parameters where possible - make sure they are globally unique where needed

Reply:
added defaultValue for parameters wherever possible.